lib/rubocop/cop/rspec/change_without_expect.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks for `change` matcher usage without an `expect` block.
+      #
+      # This cop only considered a matcher if it is chained with
+      # `from`, `by`, `by_at_least`, or `by_at_most`.
+      #
+      # @example
+      #   # bad
+      #   it 'changes the count' do
+      #     change(Counter, :count).by(1)
+      #   end
+      #
+      #   # bad
+      #   it 'changes the count' do
+      #     change(Counter, :count).by_at_least(1)
+      #   end
+      #
+      #   # bad
+      #   it 'changes the count' do
+      #     change(Counter, :count).by_at_most(1)
+      #   end
+      #
+      #   # bad
+      #   it 'changes the count' do
+      #     change(Counter, :count).from(0)
+      #   end
+      #
+      #   # bad
+      #   it 'changes the count' do
+      #     change(Counter, :count).from(0).to(1)
+      #   end
+      #
+      #   # good - no chained matchers
+      #   it 'changes the count' do
+      #     change(Counter, :count)
+      #   end
+      #
+      #   # good - ignore not change matcher
+      #   it 'changes the count' do
+      #     change(Counter, :count).from(0).by(1)
+      #   end
+      #
+      #   # good - within expect block
+      #   it 'changes the count' do
+      #     expect { subject }.to change(Counter, :count).by(1)
+      #   end
+      #
+      class ChangeWithoutExpect < RuboCop::Cop::Base
```

Comment:
Even though this would detect a number of offences, wouldn’t it make more sense to make a more broad cop that would detect all built-in block matchers? “output”, “raise_error”, …